docker/docker-compose.debian12.yaml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+
+  test-setup:
+    image: swiftly:debian12-test
+    build:
```

Comment:
should this specify the newly added Dockerfile?

Reply:
Since we've moved to the new GitHub Actions infrastructure I've removed the old docker compose files and Dockerfiles from here.